src/ui.form-label/storybook/stories.ts

```diff
@@ -108,10 +109,19 @@ Error.args = { error: "Error description" };
 export const Disabled = DefaultTemplate.bind({});
 Disabled.args = { disabled: true };
 
+export const SlotLabel = DefaultTemplate.bind({});
+SlotLabel.args = {
+  slotTemplate: `
+    <template #label>
+      <ULink label="Link component label" color="green" />
```

Comment:
better to place inside UBadge for example and set to it label from ULabel component